src/lib/settings/SettingsFolder.js

```diff
@@ -198,6 +198,7 @@ class SettingsFolder extends Map {
 		// Queue updates
 		const results = [];
 		for (const [path, entry] of values) {
+			// TODO(UnseenFaith | kyranet): This has to be updated and use group instead
```

Comment:
Follow discord.js pattern and use Object#equals(otherObject) (which would use strictEqual) or just expose strictEqual, then if group do equal comparison